src/Auth0Client.ts

```diff
@@ -938,7 +938,7 @@ export class Auth0Client {
           : await this._getTokenFromIFrame(getTokenOptions);
 
         await this.cacheManager.set({
-          client_id: this.options.client_id,
+          clientId: this.options.clientId,
```

Comment:
This will put `clientId` in our cache, while previously it was `client_id`, this would make old cached values return `undefined` for `clientId` and breaking it for our users. Even though with a new major we can add breaking changes, we should try and avoid breaking it for the end-user (try and avoid logging users out with our breaking changes) 

I think we should be able to either:
- As the cache is also using snake_case, keep the client_id in `CacheEntry` to be snake_case
- Keep it camelCased, but ensure it gets mapped to snake_case again before putting in cache. If we would do that, I think we should also camelCase all other properties on the `cacheEntry`.

Reply:
As discussed, moved the cache back to using `client_id`